### DISCUSSION TITLE
Azure Devops to Deploy IOS Application on Test-flight

### DISCUSSION THREAD

**Discussion by madcwb** [fastlane/fastlane#20222] [General]
I am using ["Azure DevOps Extension for the Apple App Store"](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store) to publish my app

But in process return error "/Users/runner/.gem-cache/gems/fastlane-2.205.2/spaceship/lib/spaceship/connect_api/token.rb:71:in `initialize': [!] string contains null byte (ArgumentError)" 
and I cant find the reason.

I need help to identify the reason.

See details

![Screenshot_5](https://user-images.githubusercontent.com/11164036/165589804-97df99a6-f579-47f3-a7cf-cec7514f79ba.png)

![Screenshot_6](https://user-images.githubusercontent.com/11164036/165589904-3de22c16-21c0-4b89-b0c1-3a951376b343.png)

**Comment by Thompsons21** [comment]
Do you use AppCenter, Azure DevOps or just do a manual deployment? Hope to get some good answers as I know things change with time and we are looking for the most simple way to do deployment even if it means doing it manually and following a list of steps. [Tell Culvers](https://www.tellculvers.onl/)

**Reply by madcwb** [reply]
I'm not using AppCenter, I'm using  manual deployment.
I follow the steps from https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store

**Reply by Thompsons21** [reply]
Great, Let me see and then tall you..
